### PR TITLE
Removed concentrated liquidity feature flag from repo and its uses al…

### PR DESCRIPTION
**Pull request by ramishj**
## What is the purpose of the change:

This PR address the issue https://github.com/osmosis-labs/osmosis-frontend/issues/3359

### Linear Task

NA

## Brief Changelog

Removes the concentrated liquidity feature flag and its usages.

## Testing and Verifying

This change has been tested locally by rebuilding the website and verified content and links are expected

### CODE REVIEW

**Comment by ramishj** [comment]
Someone see this

**Comment by ramishj** [comment]
I would like some mod to approve this....
Thanks

**Comment by MaxMillington** [comment]
@kamal-sutra @CryptoAssassin1 can you test once the build is finished?

**Comment by kamal-sutra** [comment]
@ramishj Getting this randomly
image

**Comment by ramishj** [comment]
> @ramishj Getting this randomly image

Could you please rebuild and try , if still persists ,then can you tell me how to reproduce it
Thanks

**Comment by kamal-sutra** [comment]
@ramishj Occurring randomly. Go to pools page first, refresh it and click on Portfolio page.
[Screencast from 20-06-24 08:18:56 PM IST.webm](https://github.com/osmosis-labs/osmosis-frontend/assets/116148606/e7bb5745-31a1-4ea1-abf9-73092459c617)

**Comment by ramishj** [comment]
> @ramishj Occurring randomly. Go to pools page first, refresh it and click on Portfolio page. [Screencast from 20-06-24 08:18:56 PM IST.webm](https://github.com/osmosis-labs/osmosis-frontend/assets/116148606/e7bb5745-31a1-4ea1-abf9-73092459c617)

Not happening on my side i have updated a logic , please could you rebuild and check

**Comment by kamal-sutra** [comment]
@ramishj I could repro more frequently now. We are on Chrome, Ubuntu

**Comment by CryptoAssassin1** [comment]
I can reproduce this error.

**Comment by kamal-sutra** [comment]
@ramishj happening on this preview too
https://osmosis-frontend-gwanuwo7f-osmo-labs.vercel.app/portfolio

**Comment by ramishj** [comment]
> @ramishj I could repro more frequently now. We are checking on Chrome, Ubuntu

is it still happening in the latest deployment?

**Comment by ramishj** [comment]
> @ramishj happening on this preview too https://osmosis-frontend-gwanuwo7f-osmo-labs.vercel.app/portfolio

don't know dude unable to reproduce this error

**Comment by CryptoAssassin1** [comment]
@ramishj Im not getting the error in latest build.

File: packages/web/components/complex/assets-page-v1.tsx [hunk 1]
```diff
@@ -268,14 +268,6 @@ const PoolCardsDisplayer: FunctionComponent<{ poolIds: string[] }> = observer(
 
         const apr =
           poolBonding.highestBondDuration?.aggregateApr ?? new RatePretty(0);
-
-        if (
-          !pool ||
-          (pool.type === "concentrated" && !flags.concentratedLiquidity)
```
Comment (jonator) [inline packages/web/components/complex/assets-page-v1.tsx:274]: Should prob still keep the check for pool. Just remove the statement for the CL flag

**Review by jonator — CHANGES_REQUESTED** [review]

**Comment by UkuLoskit** [comment]
@jonator @MaxMillington I made the same changes in my PR a day before, why is this PR getting precedence?

**Comment by CryptoAssassin1** [comment]
@ramishj can reproduce this error again.
Steps to reproduce this error.
1:) Land on swap page connect wallet
2:) Go to pools page and do cntrl+shift+r (make sure you have liquidity in some pools)
3:) Go to portfolio page
Note: For accounts without any pools, this error is not happening

https://github.com/osmosis-labs/osmosis-frontend/assets/103904125/e6ad4f00-9a6c-47e8-b2cc-865289ac7d6a

File: packages/web/components/complex/assets-page-v1.tsx (hunk 1 above)
Comment (ramishj) [reply packages/web/components/complex/assets-page-v1.tsx:274]: I have reverted and only removed the flag @jonator can you try rebuilding it.

**Comment by ramishj** [comment]
@CryptoAssassin1  @kamal-sutra  could you please once test it on the latest deployment

File: packages/web/components/complex/assets-page-v1.tsx (hunk 1 above)
Comment (ramishj) [reply packages/web/components/complex/assets-page-v1.tsx:274]: @jonator check

**Comment by kamal-sutra** [comment]
No error on the latest one

**Comment by ramishj** [comment]
> No error on the latest one

Approve it then mate lets merge it

**Comment by MaxMillington** [comment]
@ramishj another dev opened a working PR before you, so I'm closing this one. Thanks!

**Pull request closed without merging**